<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RelationTemplateDialog</class>
 <widget class="QDialog" name="RelationTemplateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relation templates editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/appIcon.svg</normaloff>:/resources/appIcon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,0,1">
   <item>
    <widget class="QTreeWidget" name="templateWidget">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="addDefaults">
       <property name="text">
        <string>Add defaults</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/relation.svg</normaloff>:/resources/relation.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/save.svg</normaloff>:/resources/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/trash.svg</normaloff>:/resources/trash.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addNewButton">
       <property name="text">
        <string>Add new</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/plus.svg</normaloff>:/resources/plus.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="titleEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="generationDifferenceLabel">
         <property name="toolTip">
          <string>First generation - second generatio is the generation difference.</string>
         </property>
         <property name="text">
          <string>Generation difference:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="generationDifference">
         <property name="minimum">
          <number>-20</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="descriptionEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="traitLabel">
       <property name="text">
        <string>Trait:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="trait"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="firstNameEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="secondNameLabel">
         <property name="text">
          <string>Second name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondNameEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>templateWidget</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>addNewButton</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>trait</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
